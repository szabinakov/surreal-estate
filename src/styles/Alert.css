
.Alert.alert-success {
    background-color: rgb(149, 228, 149);
    border-radius: 3%;
    margin: 6px;
    color: rgb(0, 63, 0);
}

.Alert.alert-error {
        background-color: rgb(245, 131, 131);
        border-radius: 3%;
        margin: 6px;
        color: rgb(105, 0, 0);
}