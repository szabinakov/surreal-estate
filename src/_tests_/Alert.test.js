import React from 'react'
import { render } from '@testing-library/react';
import Alert from '../components/Alert';


describe('Alert', () => {

    it('checks there is error message rendered on the site in case of an error', () => {
        const {getByText, asFragment} = render(<Alert message='Error!'/>)
        expect(getByText(/Error/).textContent).toBe('Error!')
        expect(asFragment()).toMatchSnapshot()
    })
    it('checks there is a success message rendered on the site in case of success', () => {
        const {getByText, asFragment} = render (<Alert message='Success!' success/>)
        expect(getByText(/Success/).textContent).toBe('Success!')
        expect(asFragment()).toMatchSnapshot()  

    })
})