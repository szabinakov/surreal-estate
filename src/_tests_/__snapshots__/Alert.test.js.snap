// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert checks there is a success message rendered on the site in case of success 1`] = `
<DocumentFragment>
  <div
    class="Alert alert-success"
  >
    Success!
  </div>
</DocumentFragment>
`;

exports[`Alert checks there is error message rendered on the site in case of an error 1`] = `
<DocumentFragment>
  <div
    class="Alert alert-error"
  >
    Error!
  </div>
</DocumentFragment>
`;
